content/api/hub/sample_hugo.md

```diff
@@ -0,0 +1,26 @@
+title = "Template repository for running Hugo sites"
```

Comment:
```suggestion
title = "Spin Hugo"
```
Just shortening the title so that it displays succinctly on the Spin Up Hub's card in the UI. Basing this off some other existing templates i.e. Spin NextJS that Radu created. Also, your description is excellent and clarifies all of the other  facts.